src/framework/extensions/extensionstypes.h

```diff
@@ -61,6 +50,20 @@ static inline std::string typeToString(const Type& type)
     return std::string();
 }
 
+/*
+manifest.json
+{
+
+"uri": String,                    // Example: musescore://module/target/name
+"type": String,                   // Values: form
+"title": String,                  //
+"apiversion": String              // Optional default 1
+"enabled": Boolean,               // Optional default true
+"visible": Boolean,               // Optional default true
+
+"qmlFile": String                 // Path (name) of qml file
```

Comment:
I think we need to clarify here: 
qmlFilePath: Relative path of qml file

Reply:
ok, I'll fix it in the next PR